tests/test_collector.py

```diff
@@ -21,6 +21,11 @@ def fetchMetricsSuccessMock(conn, query):
             {"database": "test", "user": "marco", "cl_active": 1, "cl_waiting": 2, "sv_active": 3, "sv_idle": 4, "sv_used": 5, "sv_tested": 6, "sv_login": 7, "maxwait": 8 },
             {"database": "prod", "user": "marco", "cl_active": 8, "cl_waiting": 7, "sv_active": 6, "sv_idle": 5, "sv_used": 4, "sv_tested": 3, "sv_login": 2, "maxwait": 1 }
         ]
+    elif query == "SHOW DATABASES":
```

Comment:
What's missing is a test on these fixtures. For example you could add a test like this:

```
    def testShouldExportDatabasesMetrics(self):
        config = PgbouncerConfig({})
        collector = PgbouncerMetricsCollector(config)
        collector._createConnection = MagicMock(return_value=False)
        collector._fetchMetrics = MagicMock(side_effect=fetchMetricsSuccessMock)

        metrics = getMetricsByName(collector.collect(), "pgbouncer_databases_database_pool_size")
        self.assertEqual(len(metrics), 2)
        self.assertEqual(metrics[0]["type"], "gauge")
        self.assertEqual(metrics[0]["value"], 50)
        self.assertEqual(metrics[0]["labels"], {"database":"testpool"})
        self.assertEqual(metrics[1]["type"], "gauge")
        self.assertEqual(metrics[1]["value"], 90)
        self.assertEqual(metrics[1]["labels"], {"database":"prodpool"})

        # TODO test reserve_pool
        # TODO test current_connections
```